datamodel/initdb.d/05_reference_implementation_support.sql

```diff
@@ -595,6 +595,12 @@ CREATE TABLE dcsa_im_v3_0.ops_event_timestamp_definition (
 CREATE INDEX ON dcsa_im_v3_0.ops_event_timestamp_definition (timestamp_definition);
 CREATE INDEX ON dcsa_im_v3_0.ops_event_timestamp_definition (payload_id);
 
+
+-- DDT-1110/1127
+ALTER TABLE dcsa_im_v3_0.operations_event
+    ADD vessel_draft integer NULL,
+    ADD miles_remaining_to_destination integer NULL;
+
```

Comment:
I personally think this should go into 03 and be a part of the official solution - but I acknowledge that it has been decided to go here for now...

Reply:
We can modify later if needed